op-challenger/game/fault/types/types_test.go

```diff
@@ -32,17 +34,17 @@ func TestIsRootPosition(t *testing.T) {
 	}{
 		{
 			name:     "ZeroRoot",
-			position: NewPositionFromGIndex(0),
+			position: NewPositionFromGIndex(common.Big0),
 			expected: true,
 		},
 		{
 			name:     "ValidRoot",
-			position: NewPositionFromGIndex(1),
+			position: NewPositionFromGIndex(big.NewInt(1)),
```

Comment:
common.Big1 and common.Big2 also exists. I'm fine using either the common / the NewInt here, but it would be nice to consistent.

Reply:
It'll be inconsistent either way since we haven't used `common.Big0` before - something Adrian asked to use.

Reply:
We could use `big.NewInt(0)` instead of `common.Big0` - I'm not particularly concerned with either, just don't want to use `new(big.Int)` to mean 0.